lib/buffer.js

```diff
@@ -21,7 +21,8 @@
 
 'use strict';
 
-const { Math, Object } = primordials;
+const { Object: { defineProperties, defineProperty, setPrototypeOf, create } } = primordials;
```

Comment:
Did you run `make lint`? It would usually complain about a line longer than 80 characters.

Reply:
Good afternoon 😃
Oh what a quick feedback!
Thank you♥️
I'll fix it when I'm back home.

Reply:
One more thing to ask for help, what's the correct way to do `prettify`? Normally I have `prettier` set up with `eslint`, but I didn't find similar in the `Makefile`.

Reply:
`make lint-js-fix` :)

Reply:
Oh I mean, for the `max-len` rule, the `eslint --fix` cannot help. 
So I have to correct it by myself right?